### ISSUE TITLE
ERROR:  Failed with exit code 134.  (rc=34304) 

### ISSUE THREAD

**Issue by bluessherpa** [marbl/canu#518]
Hi,
I am having an issue with canu.  I used Homebrew to install on my Mac. (10.12.5) 128 GB memory, 4 TB HD  space.  I had a 4 Core processor in the machine and canu would work, but would take a while to do the assembly.  I upgraded the processor and re installed Canu.  this is the output.  I also have another mac that I upgraded the processors on with a clean restart of the OS, Xcode, and install of home-brew and Canu.  I am getting the same error on that machine too.

```
bash-3.2$ canu -assemble -p b3d -d test1 genomeSize=135m -nanopore-raw firstrun.fasta
--   Reason: image not found
-- Detected Java(TM) Runtime Environment '1.8.0_101' (from 'java').
-- Detected gnuplot version '5.0 patchlevel 6' (from 'gnuplot') and image format 'png'.
-- Detected 24 CPUs and 128 gigabytes of memory.
-- No grid engine detected, grid disabled.
--
-- Run   2 jobs concurrently using   64 GB and  12 CPUs for stage 'meryl'.
-- Run   2 jobs concurrently using   13 GB and  12 CPUs for stage 'mhap (cor)'.
-- Run   3 jobs concurrently using    8 GB and   8 CPUs for stage 'overlapper (obt)'.
-- Run   3 jobs concurrently using    8 GB and   8 CPUs for stage 'overlapper (utg)'.
-- Run   6 jobs concurrently using   20 GB and   4 CPUs for stage 'falcon_sense'.
-- Run  24 jobs concurrently using    4 GB and   1 CPU  for stage 'ovStore bucketizer'.
-- Run  24 jobs concurrently using    8 GB and   1 CPU  for stage 'ovStore sorting'.
-- Run   4 jobs concurrently using    6 GB and   6 CPUs for stage 'read error detection'.
-- Run  24 jobs concurrently using    2 GB and   1 CPU  for stage 'overlap error adjustment'.
-- Run   3 jobs concurrently using   42 GB and   8 CPUs for stage 'bogart'.
-- Run   6 jobs concurrently using    4 GB and   4 CPUs for stage 'GFA alignment and processing'.
-- Run   3 jobs concurrently using   42 GB and   8 CPUs for stage 'consensus'.
--
-- Generating assembly 'b3d' in '/Volumes/LaCie/minion/test1'
--
-- Parameters:
--
--  genomeSize        135000000
--
--  Overlap Generation Limits:
--    corOvlErrorRate 0.3200 ( 32.00%)
--    obtOvlErrorRate 0.1440 ( 14.40%)
--    utgOvlErrorRate 0.1440 ( 14.40%)
--
--  Overlap Processing Limits:
--    corErrorRate    0.5000 ( 50.00%)
--    obtErrorRate    0.1440 ( 14.40%)
--    utgErrorRate    0.1440 ( 14.40%)
--    cnsErrorRate    0.1440 ( 14.40%)
--
--
-- BEGIN ASSEMBLY
--
----------------------------------------
-- Starting command on Wed Jun  7 17:30:26 2017 with 1125.584 GB free disk space

    cd unitigging
    /usr/local/Cellar/canu/1.5/Darwin-amd64/bin/gatekeeperCreate \
      -minlength 1000 \
      -o ./b3d.gkpStore.BUILDING \
      ./b3d.gkpStore.gkp \
    > ./b3d.gkpStore.BUILDING.err 2>&1
sh: line 4:  1423 Abort trap: 6           /usr/local/Cellar/canu/1.5/Darwin-amd64/bin/gatekeeperCreate -minlength 1000 -o ./b3d.gkpStore.BUILDING ./b3d.gkpStore.gkp > ./b3d.gkpStore.BUILDING.err 2>&1

-- Finished on Wed Jun  7 17:30:26 2017 (lickety-split) with 1125.584 GB free disk space
----------------------------------------
ERROR:
ERROR:  Failed with exit code 134.  (rc=34304)
ERROR:
================================================================================
Don't panic, but a mostly harmless error occurred and Canu stopped.

  Reason: image not found failed with:
  gatekeeper store exists, but contains no reads
```
Can anyone help?
Thanks.

**Comment by skoren** [comment]
This looks like a library incompatibility on your system. Try the binaries we have posted on the releases page or building from source to see if that fixes it. That said, with only 4 cores it will take a while to assemble a 135m genome, probably a month or more.

**Comment by bluessherpa** [comment]
I did some digging and ran portions of the script.  I think this is where the abort trap: 6 is coming from

**Comment by bluessherpa** [comment]
I did some digging and ran portions of the script. I think this is where the abort trap: 6 is coming from

dyld: lazy symbol binding failed: Symbol not found: ___emutls_get_address
  Referenced from: /usr/local/opt/gcc/lib/gcc/6/libgomp.1.dylib
  Expected in: /usr/lib/libSystem.B.dylib

dyld: Symbol not found: ___emutls_get_address
  Referenced from: /usr/local/opt/gcc/lib/gcc/6/libgomp.1.dylib
  Expected in: /usr/lib/libSystem.B.dylib

Abort trap: 6

Any suggestions?

**Comment by skoren** [comment]
Right, as I said I think this is a library version issue with the openmp on your system vs the build. Have you tried my previous suggestions, either using our prebuilt binaries rather than the home brew ones or guiding from source?

**Comment by bluessherpa** [comment]
I did use macports. then ran the 

git clone https://github.com/marbl/canu.git
cd canu/src
make 

So far it is running.  

Sorry, but what is the number of threads? is that processor issue?

**Comment by skoren** [comment]
The number of threads is automatically set based on what is available on the system. Canu will print a preamble about how many threads it will use for each step and how many jobs it will run in parallel at a time.

**Comment by skoren** [comment]
Closing since building from source solved the issue. Open a new issue if you encounter an error during the run.
